helmfile/values/fluentd-wc.yaml.gotmpl

```diff
@@ -46,31 +46,6 @@ extraConfigMaps:
       </log>
     </system>
 
-  10-kube-audit.conf: |-
```

Comment:
This should be kept here but removed from fluentd-user :D

Reply:
Ooh, right. Ofcorse